scripts/create-sharded-conf.py

```diff
@@ -23,23 +40,26 @@ def generate_keys(base_dir, shard_count, helper_name_format):
                 "--tls-key",
                 f"{dir_path}/{helper_name}.key",
             ]
+
             if j == 0:  # Only add mk-public-key and mk-private-key params for the first shard
                 keygen_cmd.extend([
                     "--mk-public-key",
                     f"{dir_path}/{helper_name}_mk.pub",
                     "--mk-private-key",
                     f"{dir_path}/{helper_name}_mk.key",
                 ])
+
             subprocess.run(keygen_cmd)
 
             # Copy mk public and private keys from the first shard to other shards
             if j != 0:
-                src_dir = f"{base_dir}/helper{i}/shard{0}"
+                src_dir = f"{base_dir}/helper{i}/shard0"
```

Comment:
I tried to trace all references to the shard directory naming convention in the Rust code and replace them with function call. We can't do it here, but it will be great if we had a comment here explaining that this needs to be kept in sync with the Rust code (ideally with code pointers https://github.com/private-attribution/ipa/blob/d8b73fb2521c8f90a8fd9897d462a5a9816adeb7/ipa-core/src/cli/clientconf.rs#L228)